eslint.config.mjs

```diff
@@ -43,10 +43,9 @@ export default [
       '**/node_modules/**',
       'benchmark/fixtures/**',
       'benchmark/tmp/**',
-      'doc/**/*.js',
       'doc/changelogs/CHANGELOG_V1*.md',
-      '!doc/api_assets/*.js',
       '!doc/changelogs/CHANGELOG_V18.md',
+      'doc/changelogs/CHANGELOG_V23.md',
```

Comment:
AFAIK we shouldn't make manual changes in changelogs, but if there's still a window to adjust `CHANGELOG_V23` before v24.x release, we can add the ` //`s here:
https://github.com/nodejs/node/blob/746b17e1a5a9d09294b4c993a0319be6e5c39837/doc/changelogs/CHANGELOG_V23.md?plain=1#L179-L183

Reply:
We have precedent for this (e.g. https://github.com/nodejs/node/commit/c6dabe30832e40dab7fa35fd2ccd0135c74ca091), AFAIK there are no issues with modifying the changelogs after the fact (it's not like someone is going to compare the hashes of the CHANGELOG files between `main` and the release tag)